docs/services/slam/_index.md

```diff
@@ -152,27 +152,6 @@ pcd_map = await slam_svc.get_point_cloud_map()
 Get the internal state of the SLAM algorithm required to continue mapping/localization.
 
 {{< tabs >}}
```

Comment:
reordering since we generally have python first

Reply:
@npentrel Could you reorder the first methods on this page too? It looks like a few still have python second

Reply:
done